### PR TITLE
docs: add supported architectures

### PR DESCRIPTION
**Pull request by codingben**
**What this PR does / why we need it**:

Add a list of containerdisks and supported
architectures.

Signed-off-by: Ben Oukhanov <boukhanov@redhat.com>

**Release note**:

```release-note
None
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # KubeVirt curated Containerdisks
 
+| Name               | Architecture       |
+| ------------------ | ------------------ |
+| CentOS Stream      | amd64, arm64       |
+| Fedora             | amd64, arm64       |
+| CentOS             | amd64              |
+| Ubuntu             | amd64, arm64       |
```
Comment (lyarwood) [inline README.md:8]: Can you link to the repo for each?

```suggestion
| [CentOS Stream](https://quay.io/containerdisks/centos-stream)     | amd64, arm64       |
```

Comment (codingben) [reply README.md]: Done.

**Pull request merged**